frontend-shared/README.md

```diff
@@ -0,0 +1,27 @@
+Hypothesis frontend shared
```

Comment:
I would suggest to use `# Heading` syntax for all markdown headings (note that I mean always use 1+ hashes followed by a space followed by the heading, not always using exactly one hash). This makes it easier to tell the relative heading level of different headings as you can simply count the number of `#`s.